### DISCUSSION TITLE
Beta 0.9.10

### DISCUSSION THREAD

**Discussion by monee121** [robertodoering/harpy#764] [General]
Hello there! I just want to say I recently got the update on Pro version. I love the link preview! It reminds me of a link preview on either FB or in Outlook emails. Also I finally can open links internally. Previously even with the workaround it wasn't working for me. Now it's smooth as butter, thank you for enabling by default. Love this app!

**Comment by robertodoering** [comment]
Thank you! I'm glad you like the update 🚀

**Comment by scottiefalkon** [comment]
Perhaps someone can help me with the API key. I follow the directions, but
I don't get the option for the three-legged access point. It also asks me
to provide a website name.

**Reply by robertodoering** [reply]
The website name doesn't matter, you can enter whatever you want.
The settings page changed a bit since writing the guide. You no longer need to explicitly enable three-legged auth. Instead you can just enter the callback url `harpy://` in the api settings.

<img src="https://user-images.githubusercontent.com/20045287/191509957-1c94ccd4-245d-4277-82ca-bde41cf0444a.png" width=350/>

I'll update the setup guide to better reflect the new API settings UI.

**Reply by killerlearner** [reply]
> The website name doesn't matter, you can enter whatever you want. The settings page changed a bit since writing the guide. You no longer need to explicitly enable three-legged auth. Instead you can just enter the callback url `harpy://` in the api settings.
> 
> <img alt="" width="350/" src="https://user-images.githubusercontent.com/20045287/191509957-1c94ccd4-245d-4277-82ca-bde41cf0444a.png">
> 
> I'll update the setup guide to better reflect the new API settings UI.

Will essential api work or we absolutely need elevated api for personal use.

**Comment by scottiefalkon** [comment]
Thanks a bunch.

**Comment by killerlearner** [comment]
@robertodoering  just tried login with my api but i am unable to login. So is elevated developer account needed for using our own api, can't use essential account 🥲. Cuz I don't know how to update my account in elevated.

**Reply by robertodoering** [reply]
Yes, see https://github.com/robertodoering/harpy/wiki/Twitter-api-key-setup

**Reply by killerlearner** [reply]
Got any suggestions how to get account accept as elevated. I tried but got rejected ಠ⁠_⁠ಠ
